Add documentation for configuration

umireon opened the pull request:
A new user of ec2c might be confused with README because it does not have any description about credentials.

This PR will add a small description about configuring credentials.

umireon committed:
Add documentation about ~/.aws/credentials

umireon committed:
Mention to specify the region

dtan4 commented:
Please describe about environment variables too, like [this](https://github.com/dtan4/s3url/tree/0ebc096229e46f560827b9e041479abf5cf3823b#usage) 💡 

```bash
export AWS_ACCESS_KEY_ID=XXXXXXXXXXXXXXXXXXXX
export AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
export AWS_REGION=xx-yyyy-0
```

umireon committed:
Revise the document of configuration

The document was changed to follow the aws-sdk-go documentation.
https://github.com/aws/aws-sdk-go#configuring-credentials

umireon commented:
The documentation was revised according to [the configuration document of aws-sdk-go](https://github.com/aws/aws-sdk-go#configuring-credentials).

dtan4 commented:
LGTM 👍 👍 👍 Thanks ❗️
